.github/e2e/tls-job.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: dap-secret-webhook-certgen
+  namespace: flyte
+spec:
+  template:
+    metadata:
+      name: dap-secret-webhook-certgen
+    spec:
+      serviceAccountName: dap-secret-webhook
+      restartPolicy: Never
+      containers:
+      - name: certgen
+        image: "quay.io/reactiveops/ci-images:v11-alpine"
+        command: ["bash"]
+        args:
+          - -c
+          - |
+            # Generates the a CA cert, a server key, and a server cert signed by the CA.
+            # reference:
+            # https://github.com/kubernetes/kubernetes/blob/master/plugin/pkg/admission/webhook/gencerts.sh
+            set -o errexit
+            set -o nounset
+            set -o pipefail
+            apk --update add openssl
+            CN_BASE="dap-secret-webhook_webhook"
```

Comment:
On a second thought, I think it should be part of the helm chart scope

Reply:
We can enhance it further by letting the webhook create the cert base on config (also made available in helm). For now the cert creation is outside of the webhook/helm scope.

For using self-sign cert, this flag can be on for the webhook to generate the cert.
For using signed cert (or any other), it can be attached via env var as per currently support

Reply:
nvm i misunderstood the intent, will put the job as part of the helm chart